packages/suite/src/reducers/wallet/coinjoinReducer.ts

```diff
@@ -126,3 +135,35 @@ export const coinjoinReducer = (
             // no default
         }
     });
+
+export const selectCoinjoinAccounts = (state: CoinjoinRootState) => state.wallet.coinjoin.accounts;
+
+export const selectCoinjoinAccountByKey = createSelector(
+    [selectCoinjoinAccounts, (_state: CoinjoinRootState, accountKey: string) => accountKey],
+    (accounts, accountKey) => accounts.find(account => account.key === accountKey),
+);
+
+export const selectAccountCoinjoinInfo = createSelector(
+    [
+        (state: CoinjoinRootState, accountKey: string) =>
+            selectCoinjoinAccountByKey(state, accountKey),
+        (state: AccountsRootState, accountKey: string) => selectAccountByKey(state, accountKey),
+    ],
+    (coinjoinAccount, relatedAccount) => {
+        const { targetAnonymity, session: currentSession } = coinjoinAccount || {};
+        const { addresses, utxo: utxos } = relatedAccount || {};
+
+        const fundsBreakdown = breakdownCoinjoinFunds({
+            targetAnonymity,
+            anonymitySet: addresses?.anonymitySet,
+            utxos,
+            registeredUtxos: currentSession?.registeredUtxos,
+        });
+
+        return {
+            fundsBreakdown,
```

Comment:
Are there any other place where this could be useful except FundsPrivacyBreakdown component? If not, maybe it makes sense to move the breakdown there?

Reply:
Yes, the coin selection in send form.

Reply:
Also, since it's not a hook anymore, I was actually thinking about selecting a session and anonymity level separately.

Reply:
consider breaking this selector to 3 selectors

Reply:
https://github.com/trezor/trezor-suite/pull/6508/commits/a59e87f0ce29ea5a7e471582ad209004d273d3df